### PR TITLE
Integration with TensorFlow Tensors

### PR DESCRIPTION
**Pull request by pdet**
We can now directly produce tensorflow tensors from a DuckDB query.

e.g.,
``` con.execute("select * from t").tf() ```

### CODE REVIEW

File: tools/pythonpkg/duckdb-stubs/__init__.pyi [hunk 1]
```diff
@@ -136,6 +136,7 @@ class DuckDBPyConnection:
     def load_extension(self, extension: str) -> None: ...
     def pl(self, chunk_size: int = ..., connection: DuckDBPyConnection = ...) -> polars.DataFrame: ...
     def torch(self, connection: DuckDBPyConnection = ...) -> dict: ...
+    def tf(self, connection: DuckDBPyConnection = ...) -> dict: ...
```
Comment (Mause) [inline tools/pythonpkg/duckdb-stubs/__init__.pyi:139]: Is tf a well known shortening of tensor flow? My first thought at seeing it is "transform"

**Review by Mause — COMMENTED** [review]
Looks good!

File: tools/pythonpkg/duckdb-stubs/__init__.pyi (hunk 1 above)
Comment (pdet) [reply tools/pythonpkg/duckdb-stubs/__init__.pyi:139]: Im no ML guy but from their tutorials it seems like a common alias 😅

Comment (Tishj) [reply tools/pythonpkg/duckdb-stubs/__init__.pyi:139]: I can imagine in the ecosystem that uses
`import pandas as pd`
`import numpy as np`
that `import tensorflow as tf` is a commonly used acronym

**Comment by Mytherin** [comment]
Thanks!

**Comment by mullerhai** [comment]
@pdet  we also need duckdb support pytorch tensor generate ,does it support?

**Pull request merged**